src/main/java/com/easypost/model/Report.java

```diff
@@ -40,22 +39,16 @@ public static Report create(final Map<String, Object> params) throws EasyPostExc
      * @throws EasyPostException when the request fails.
      */
     public static Report create(final Map<String, Object> params, final String apiKey) throws EasyPostException {
-        String type = (String) params.get("type");
-
-        Map<String, Object> wrappedParams = new HashMap<String, Object>();
-        wrappedParams.put("report", params);
-        wrappedParams.put("start_date", params.get("start_date"));
-        wrappedParams.put("end_date", params.get("end_date"));
+        if (params.containsKey("type")) {
+            String type = (String) params.get("type");
```

Comment:
Since this is not reused, we can simply pass in `params.get("type")` to the `reportURL` param below.